apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -255,6 +378,33 @@ const InfiniteScrollContainer = (props: InfiniteScrollContainerPropsInterface):
                 { loading && (
                     <LoaderPlaceholder />
                 ) }
+                { !loading && logType === TabIndex.AUDIT_LOGS && (
+                    <div className="log-header">
+                        <div className="log-Recordedtime">
+                            <Typography variant="body1">
+                                {
+                                    t("extensions:develop.monitor.logView.headers.recordedAt")
+                                }
```

Comment:
Let's fix the formatting in other places as well. 

```suggestion
                                { t("extensions:develop.monitor.logView.headers.recordedAt") }
```